shell/edit/resources.cattle.io.backup.vue

```diff
@@ -245,6 +326,57 @@ export default {
           </div>
         </div>
       </div>
+      <div class="bordered-section">
+        <div class="row mb-10">
+          <div class="col span-12">
+            <h3>{{ t('backupRestoreOperator.backup.label') }}</h3>
+            <div>{{ t('backupRestoreOperator.backup.description') }}</div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col span-12">
+            <RadioGroup
+              v-model:value="resourceSetSelection"
+              name="resourceSet"
+              :options="resourceSetOptions"
+              :mode="mode"
+            />
+          </div>
+        </div>
+        <div
+          v-if="showCustomResourceSetOptions"
+          class="row mt-10"
+        >
+          <div class="col span-6">
+            <LabeledSelect
+              v-model:value="value.spec.resourceSetName"
+              :mode="mode"
+              :options="customResourceSetOptions"
+              :label="t('backupRestoreOperator.backup.resourceSetOptions.customResourceSetLabel')"
+            />
+          </div>
+        </div>
+        <div
+          v-if="showEncryptionWarningBanner"
+          class="row mt-10"
+        >
+          <div class="col span-12">
+            <Banner color="warning">
+              <span v-clean-html="t('backupRestoreOperator.backup.enableEncryptionWarning')" />
+            </Banner>
+          </div>
+        </div>
+        <div
+          v-if="showMissingResourceSetWarningBanner"
+          class="row mt-10"
+        >
+          <div class="col span-12">
+            <Banner color="warning">
+              <span v-clean-html="t('backupRestoreOperator.backup.missingResourceSetWarning', {resourceSet: value.spec.resourceSetName})" />
+            </Banner>
+          </div>
+        </div>
```

Comment:
accessibility: We'll want to add `role="alert` to each banner if they will conditionally render based on user input or any other action taken on the form.

Reply:
Added `role="alert"` to one of the two banners based on the conversation we had.